README.md

```diff
@@ -2,6 +2,8 @@
 
 [![All Contributors](https://img.shields.io/badge/all_contributors-5-orange.svg?style=flat-square)](#contributors)
 
+> :warning: This package doesn't support AsyncAPI 1.x anymore. We recommend to upgrade to the latest AsyncAPI version using the [AsyncAPI converter](https://github.com/asyncapi/converter). If you need to convert documents on the fly, you may use the [Node.js](https://github.com/asyncapi/converter) or [Go](https://github.com/asyncapi/converter-go) converters.
```

Comment:
maybe add that npm package up to some version supports v1? and after that only v2

Reply:
we can make something like a `v1` branch with old code before migration

Reply:
@aerfio We can create tag for appropriate revision, so this will be easy.